src/constants/contracts.ts

```diff
@@ -39,7 +39,7 @@ export const contracts = constantContracts({
   REGULAR_POOL_FACTORY: {
     address: {
       [Chains.mainnet]: '0x722969A3fdc778a5cC7CbC8DC8Ae3e96a288f853',
-      [Chains.goerli]: '0xb52736cE26345942134b11083e88A0DD9Ef1be5A',
+      [Chains.goerli]: '0x3068a70D28429BdE8A9D3F2ebb4dA710d6b56F39',
```

Comment:
We will need to change it again before release, right?

Reply:
This a new factory contract that I have deployed to make it with work with the new library we are using now to create the merkle tree. Current contract won't work.

Reply:
Yeah, I mean when new contracts will be deployed we will need to update it here, just checking that I understood correctly